### DISCUSSION TITLE
[Feature Request] Ability for a template to be submitted multiple times by the same user (email)

### DISCUSSION THREAD

**Discussion by NuttCorp** [docusealco/docuseal#32] [Ideas]
If I have a template that is filled out on a monthly basis by the same users it wont let the user do the template again. Cloning the template and having a new link isn't ideal.

**Comment by AlexBTurchyn** [comment]
@NuttCorp Your suggestion really makes sense. I will try to add this feature in one of the next releases

**Reply by Jpow1987** [reply]
I am looking for this feature as well. has this been added?

**Reply by omohokcoj** [reply]
@Jpow1987 it's possible with the 'resubmit' feature/button the signer sees on the completed submission page

**Reply by Jpow1987** [reply]
I see that. Is there a way to get it where it doesn't look for previous completion of the submitter?

**Reply by omohokcoj** [reply]
@Jpow1987 can you please describe your use case and the problem you're trying to solve for your use case?

**Reply by Jpow1987** [reply]
So it would be like an inspection form, where each time the device would be inspected, another form would be filled out and signed. I know the resubmit button is there, but it would be great if the technician doing it could just go to the form link and fill it out and submit it without it checking if it was filled out previously by his email.

**Reply by omohokcoj** [reply]
@Jpow1987 thanks for providing those details, unfortunately resubmit button is the only way to submit the new form if it was submitted previously. Submitting again without the resubmit button requires API or Embedding

**Reply by touro411** [reply]
Hi, I have "Allow to resubmit completed forms" in settings selected. If I copy the link, I cannot complete the document with the same emailaddress. I have no button to do the signung process again. Whats the reason for this behavior?

**Reply by omohokcoj** [reply]
@touro411 only template forms with 1 signing party can be resubmitted

**Reply by f3st3r113** [reply]
Not sure where this request stands, however, our use case is the following:

We have users that attend training sessions, they utilize a form that requires them to enter the class information, then submit to a training advisor for approval. Once approved (signed), it gets sent back. 

They may have multiple classes that they sign up for, so being able to use the form more than once is vital. 

Have we come up with a solution yet?

**Reply by omohokcoj** [reply]
@f3st3r113 set default recipient details for the "advisor" party so that the template can be resubmitted by the first party:
https://www.docuseal.com/resources/pre-filling-recipients

**Reply by f3st3r113** [reply]
We have it set up so that when a user fills out the form, it will automatically get sent to the approver. We want the user to just be able to open the form, fill it out, and then it gets sent.

What I did determine is that if we archive, the user can then resubmit without issue.
